models/src/anemoi/models/models/encoder_processor_decoder.py

```diff
@@ -277,7 +277,7 @@ def _run_mapper(
             use_reentrant=use_reentrant,
         )
 
-    def forward(self, x: Tensor, model_comm_group: Optional[ProcessGroup] = None) -> Tensor:
+    def forward(self, x: Tensor, *, model_comm_group: Optional[ProcessGroup] = None, **kwargs) -> Tensor:
```

Comment:
Be careful: This line is a breaking change, which is incompatible with `anemoi-training==0.4.0`. Please consider bumbing the minor version number.

Reply:
... Nonetheless, I believe this kind of change is quite beneficial for the code readability. :+1: 

We've just noticed that if you `pip install anemoi-training=0.4.0` (or even `pip install anemoi-training`) at the moment, pip will install `anemoi-models=0.5.1` which includes this incompatible change leading to non-functional installation of anemoi. (This could also be partially responsible for breaking the downstream CI right now.)